APTITUDE/APTITUDETEST.Data/Data/CandidateRepository.cs

```diff
@@ -595,9 +595,6 @@ public async Task<JsonResult> GetQuestionsStatus(int userId)
                     int answered = 0;
                     int unAnswered = 0;
                     bool isQuestionsMenu = (bool)test?.IsQuestionsMenu;
```

Comment:
is this variable required now ?

Reply:
Yes as we are passing the same to frontend to determine if we need to display menu or not